pkg/webui/console/views/application-general-settings/index.js

```diff
@@ -126,6 +127,7 @@ const validationSchema = Yup.object().shape({
       sharedMessages.attributeValueValidateTooLong,
       attributeValueTooLongCheck,
     ),
+    skip_payload_crypto: Yup.boolean().default(false),
```

Comment:
```suggestion
    skip_payload_crypto: Yup.boolean(),
```

I don't think you need that.